apps/meteor/server/routes/avatar/user.ts

```diff
@@ -72,3 +73,62 @@ export const userAvatarByUsername = async function (request: IncomingMessage, re
 
 	serveSvgAvatarInRequestedFormat({ nameOrUsername: requestUsername, req, res });
 };
+
+export const userAvatarById = async function (request: IncomingMessage, res: ServerResponse, next: NextFunction) {
```

Comment:
Why not create a 3rd function that receives user as an argument, and not duplicate the entire function?

Reply:
I can't share the functions because in some cases I don't need the user, and the logic for get the avatar by ID is slightly different. It would either be less efficient, or very convoluted.